### ISSUE TITLE
Start Using SAP HANA 2.0, express edition (Virtual Machine Method)

### ISSUE THREAD

**Issue by chruer** [SAPDocuments/Tutorials#1993]
Multiple things:
1. You should mention the default keyboard layout in the beginning such that newbies like me already consider it when entering the passwords in the first steps.
2. The PW that I entered as HANA database master PW was not accepted for XSA_ADMIN without giving any error message. It is due to the fact that different password policies are followed. I entered special characters which are not allowed for XSA_ADMIN by default policy. Resetting the master PW didn't help, I tried multiple approaches, also the one given by you in a different post. I had to reinstall the whole thing just in order to give another PW initially :-(

**Comment by JCurrie27** [comment]
Thanks for these two enhancement requests. These are now in-process and will be available in an upcoming refresh of the tutorials.

**Comment by tai-le** [comment]
Hi chruer,

What password did you use?

Which HANA, express edition version are you using?  Can you "cat /usr/sap/HXE/SYS/global/hdb/hxe_info.txt"?

Thanks.
-TL

**Comment by chruer** [comment]
Hi TL,

It's 2.00.021.00.20171013.1. I'm using VM under Windows.

**Comment by tai-le** [comment]
What special character(s) did you use in the password?  I would you like to reproduce it in-house.  Thanks.

**Comment by chruer** [comment]
I used /

**Comment by tai-le** [comment]
I am not able to reproduce the issue with "/" character in the password.

Which keyboard layout did you use when entering the password?  It's possible "/" character you typed translated into an illegal character.

**Comment by chruer** [comment]
Actually I used "-" on my German keyboard which translated into "/" in US layout. Apart from that I only used upper and lowercase letters as well as digits. The HANA password policy documentation is very clear that anything but letters and numbers would not be allowed if "special characters" is not ticked. However I was able to also use "special characters" when changing the PW subsequently even though it was not ticked.

Maybe it's a problem only when setting the master password?

However I may have used the backspace character and instead of deleting the character it was considered part of the PW in HANA? But I doubt that I used it in the same way when entering the PW for the second time.
